app.js

```diff
@@ -28,12 +28,11 @@ io.on('connection', (socket) => {
 
 	//When client requests for setting username
 	socket.on('set username', (name) => {
-		name = name.trim()
+		name = (name || "").trim()
```

Comment:
Any specific reason to change this?

Reply:
If the socket.io client not send a name (undefined), its code will break.